### PR TITLE
Add microsoft-azure-storage-connector-example

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+* `/createcontainer` : Creates a new container in the Microsoft Azure Storage account with the specified container name for store employee details.
+* `/listcontainer` : Retrieve information about the created containers from the Microsoft Azure Storage account.
+* `/adddetails`: Upload text or binary employee data (blob) and stored into the specified container.
+* `/listdetails` : Retrieve information about the added employee deta (blobs).
+* `/deletedetails` : Remove added employee data from the specified text or binary employee data (blob).
+* `/deletecontainer` : Remove created container in the Microsoft Azure Storage account.
+
+To know the further information about these operations please refer the [Microsoft Azure Storage connector reference guide](microsoft-azure-storage-connector-reference.md).
+
+> **Note**: Before invoke the API you need to create a **Storage Account** in **Microsoft Azure Storage account**, see [Azure Storage Configuration](microsoft-azure-storage-configuration.md) documentation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:29]:
```suggestion
> **Note**: Before invoking the API, you need to create a **Storage Account** in **Microsoft Azure Storage account**. See [Azure Storage Configuration](microsoft-azure-storage-configuration.md) documentation for more information.
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+All six operations are exposed via an API. The API with the context `/resources` has six resources  
+
+* `/createcontainer` : Creates a new container in the Microsoft Azure Storage account with the specified container name for store employee details.
+* `/listcontainer` : Retrieve information about the created containers from the Microsoft Azure Storage account.
+* `/adddetails`: Upload text or binary employee data (blob) and stored into the specified container.
+* `/listdetails` : Retrieve information about the added employee deta (blobs).
+* `/deletedetails` : Remove added employee data from the specified text or binary employee data (blob).
+* `/deletecontainer` : Remove created container in the Microsoft Azure Storage account.
+
+To know the further information about these operations please refer the [Microsoft Azure Storage connector reference guide](microsoft-azure-storage-connector-reference.md).
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:27]:
```suggestion
For more information about these operations, please refer to the [Microsoft Azure Storage connector reference guide](microsoft-azure-storage-connector-reference.md).
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+> **Note**: Before invoke the API you need to create a **Storage Account** in **Microsoft Azure Storage account**, see [Azure Storage Configuration](microsoft-azure-storage-configuration.md) documentation.
+
+Following diagram shows the overall solution. User creates a container, stores some text or binary employee data (blob) into the container and then he receives it back. To invoke each operation user uses the same API.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:31]:
```suggestion
The following diagram shows the overall solution. The user creates a container, stores some text or binary employee data (blob) into the container, and then receives it back. To invoke each operation, the user uses the same API. 
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+   
+   ```
+**Note :** Please modify below properties of the configuration as applicable.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:176]:
```suggestion
**Note**: Please modify the following properties of the configuration as applicable.
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
+# Microsoft Azure Storage Connector
+
+The Microsoft Azure Storage Connector allows you to access the Azure Storage services using Microsoft Azure Storage Java SDK through WSO2 EI. [Azure Storage](https://azure.microsoft.com/en-us/) is a Microsoft-managed cloud service that provides storage that is highly available, secure, durable, scalable and redundant. The Azure Storage consists of four primary Azure Storage types. They are blob storage, table storage, file storage, queue storage.
+
+Given below is a sample scenario that demonstrates how working with containers and blobs operations using the WSO2 Microsoft Azure Storage Connector.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:5]:
```suggestion
Given below is a sample scenario that demonstrates how to work with container and blob operations using the WSO2 Microsoft Azure Storage Connector.
```

Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:5]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:27]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:29]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:31]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:176]: Updated.

**Pull request merged**